model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -119,7 +119,7 @@ def _last_results_increased_throughput(self):
             lastest_throughput > prev_throughput
             for prev_throughput in self._curr_max_batch_size_throughputs[:-1])
 
-    def _get_last_results_max_throughput(self):
+    def _get_last_results_max_throughput(self) -> Union[float, None]:
```

Comment:
It looks cleaner to say Optional[float] instead of Union[float, None] and I've switched over to this in my code:
https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint

Reply:
Nice! I've boyscouted that everywhere I could find